pkg/service/init.go

```diff
@@ -266,21 +266,15 @@ func (smf *SMF) Start() {
 	}
 	udp.Run(pfcp.Dispatch)
 
-	for _, upf := range context.SMF_Self().UserPlaneInformation.UPFs {
-		var upfStr string
-		if upf.NodeID.NodeIdType == pfcpType.NodeIdTypeFqdn {
-			upfStr = fmt.Sprintf("[%s](%s)", upf.NodeID.FQDN, upf.NodeID.ResolveNodeIdToIp().String())
-		} else {
-			upfStr = fmt.Sprintf("[%s]", upf.NodeID.IP.String())
-		}
-		if err = setupPfcpAssociation(upf.UPF, upfStr); err != nil {
-			logger.AppLog.Errorf("Failed to setup an association with UPF%s, error:%+v", upfStr, err)
-		}
+	ctx, cancel := context.WithCancel(context.Background())
+	smf_context.SMF_Self().PFCPCancelFunc = cancel
```

Comment:
When to use the cancel() function?

Reply:
It is used to accommodate dynamic configuration changes that may be implemented in the future.